core/src/main/scala/com.snowplowanalytics.stream/loader/ElasticsearchSinkApp.scala

```diff
@@ -52,7 +52,7 @@ trait ElasticsearchSinkApp {
     val projectName = "snowplow-elasticsearch-loader"
     case class FileConfig(config: File = new File("."))
     val parser = new scopt.OptionParser[FileConfig](projectName) {
-      head(projectName, generated.Settings.version)
+      head(projectName, com.snowplowanalytics.stream.loader.generated.Settings.version)
```

Comment:
this shouldn't need to be added

Reply:
for some reason my code doesn't compile without it.  Maybe some sbt settings with the new laptop.

Reply:
mmh weird